corpus/templates/registration/password_reset_done.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block title %}
+    Done! | Reset Password
+{% endblock %}
+
+{% block content %}
+    <div class="card w-3/4 lg:w-1/3 mx-auto my-10 bg-base-200 shadow-xl">
+        <div class="card-body">
+            <h1 class="card-title text-2xl">Reset Password</h1>
+            <p>
+                We've emailed you instructions for setting your password, if an account exists with the email you
+                entered. You should receive them shortly. The reset link is valid for <strong>30 minutes</strong>.
```

Comment:
30 minutes is way too less for the onboarding right? Do you think we can keep it higher (maybe 24 hours) until the onboarding is done?

Once onboarding is finished I think 30 minutes is a reasonable duration for validity.